src/simtools/applications/validate_file_using_schema.py

```diff
@@ -116,6 +116,19 @@ def _get_schema_file_name(args_dict, data_dict=None):
     return schema_file
 
 
+def _get_file_list(file_directory=None, file_name=None):
```

Comment:
Maybe consider renaming the function to get_json_file_list

Reply:
Good suggestion. Done.